packages/contract-addresses/CHANGELOG.json

```diff
@@ -1,12 +1,23 @@
 [
+    {
+        "version": "3.2.0",
+        "changes": [
+            {
+                "note": "Added `getNetworkIdByExchangeAddressOrThrow`",
```

Comment:
The `getNetworkIdByExchangeAddressOrThrow` function is not actually included in this PR. AFAICT it doesn't exist in the `merge-development-into-3.0` branch. The change was made in this PR: https://github.com/0xProject/0x-monorepo/pull/2096.